src/components/MovieDetails/index.tsx

```diff
@@ -354,19 +354,23 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
           </div>
           <h1 className="text-2xl lg:text-4xl">
             {data.title}{' '}
-            <span className="text-2xl">({data.releaseDate.slice(0, 4)})</span>
+            {data.releaseDate && (
+              <span className="text-2xl">({data.releaseDate.slice(0, 4)})</span>
+            )}
           </h1>
           <span className="mt-1 text-xs lg:text-base lg:mt-0">
-            {(data.runtime ?? 0) > 0 && (
+            {data.runtime || data.genres.length ? (
               <>
-                <FormattedMessage
-                  {...messages.runtime}
-                  values={{ minutes: data.runtime }}
-                />{' '}
-                |{' '}
+                {data.runtime ? (
+                  <FormattedMessage
+                    {...messages.runtime}
+                    values={{ minutes: data.runtime }}
+                  />
+                ) : null}
+                {data.runtime && data.genres.length ? ' | ' : ''}
+                {data.genres.map((g) => g.name).join(', ')}
```

Comment:
Maybe build the string elsewhere and do a `.join(' | ')`? Especially since we might add content ratings here?

Reply:
Oh yeah that's a great idea, will make that change now 😄